### PR TITLE
docs: Document the status command

### PR DESCRIPTION
**Pull request by mortent**
Adds documentation for the `kpt live status` command.

### CODE REVIEW

**Review by phanimarupaka — APPROVED** [review]

File: site/content/en/reference/live/status/_index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+type: docs
+description: >
+   Status shows the status for the resources in the cluster
+---
+<!--mdtogo:Short
+    Status shows the status for the resources in the cluster
+-->
+
+The status command will print the status for all resources in the live state
+that belong to the current inventory. It will use the inventory template to
```
Comment (frankfarzan) [inline site/content/en/reference/live/status/_index.md:13]: you use a proper noun here: Inventory Template

Is that defined anywhere?

File: internal/docs/generated/livedocs/docs.go [hunk 2]
```diff
@@ -182,3 +182,49 @@ var PreviewExamples = `
...
+    The default value is ‘events’.
+  
+  --timeout (duration):
+    Determines how long the command should run before exiting. This deadline will
+    be enforced regardless of the value of the --poll-until flag. The default is
+    to wait forever.
+`
+var StatusExamples = `
+  # Monitor status for a set of resources based on manifests. Output in table format:
+  kpt live status my-app/ --poll-until=forever --output=table
```
Comment (frankfarzan) [inline internal/docs/generated/livedocs/docs.go:226]: nit: As a general rule, I recommend showing the simplest CLI usage first:

kpt live status my-app

and then follow up with more advanced examples like changing polling and output.

File: site/content/en/reference/live/status/_index.md (hunk 1 above)
Comment (mortent) [reply site/content/en/reference/live/status/_index.md]: Updated this with a link to the apply documentation where we explain the Inventory Template.

File: internal/docs/generated/livedocs/docs.go (hunk 2 above)
Comment (mortent) [reply internal/docs/generated/livedocs/docs.go:226]: Fixed.

File: site/content/en/reference/live/status/_index.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  The default value is ‘events’.
+
+--timeout (duration):
+  Determines how long the command should run before exiting. This deadline will
+  be enforced regardless of the value of the --poll-until flag. The default is
+  to wait forever.
+```
+<!--mdtogo-->
+
+[Inventory Template]: https://googlecontainertools.github.io/kpt/reference/live/apply/#prune
```
Comment (frankfarzan) [inline site/content/en/reference/live/status/_index.md:82]: This page doesn't actually use the name 'Inventory Template' anywhere. As a general rule, define proper nouns explicitly once and refer to it.

**Review by frankfarzan — APPROVED** [review]
Approved after suggestion addresed

File: site/content/en/reference/live/status/_index.md (hunk 3 above)
Comment (mortent) [reply site/content/en/reference/live/status/_index.md:82]: The current version doesn't, but this PR includes changes to the page that removes the old `grouping object` terminology to the current `inventory object`/`inventory template` terminology.

**Pull request merged**